### ISSUE TITLE
Fails to use globally defined resolvers for build plugins

### ISSUE THREAD

**Issue by jeantil** [alexarchambault/coursier#292]
By globally defined I mean defined in ~/.sbt/0.13/plugins/*.sbt

To reproduce try this  :

```
mkdir plugin-resolvers
echo 'name:="plugin-resolvers"' > plugin-resolvers/build.sbt
echo 'resolvers += Resolver.url("jetbrains-bintray", url("http://dl.bintray.com/jetbrains/sbt-plugins/"))(Resolver.ivyStylePatterns)'> ~/.sbt/0.13/plugins/idea-settings.sbt
echo 'addSbtPlugin("org.jetbrains" % "sbt-ide-settings" % "0.1.2")' >> ~/.sbt/0.13/plugins/idea-settings.sbt
cd plugin-resolvers
sbt
```

once in sbt run `updateSbtClassifiers`

At the moment I end up with 

```
[info] Loading global plugins from /Users/jean/.sbt/0.13/plugins
[info] Updating {file:/Users/jean/.sbt/0.13/plugins/}global-plugins...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Set current project to plugin-resolvers (in build file:/Users/jean/dev/sdev/src/temp/plugin-resolvers/plugin-resolvers/)
> updateSbtClassifiers
[info] Updating default:plugin-resolvers_2.10:0.1-SNAPSHOT (sbt classifiers)
[error]
[error] 1 error(s):
[error]   org.jetbrains:sbt-ide-settings;sbtVersion=0.13;scalaVersion=2.10:0.1.2:
[error]     not found: /Users/jean/.sbt/0.13/plugins/target/resolution-cache/org.jetbrains/sbt-ide-settings/scala_2.10/sbt_0.13/0.1.2/resolved.xml.xml
[error]     Not found
[error]     not found: /Users/jean/.ivy2/local/org.jetbrains/sbt-ide-settings/scala_2.10/sbt_0.13/0.1.2/ivys/ivy.xml
[error]     not found: https://repo1.maven.org/maven2/org/jetbrains/sbt-ide-settings_2.10_0.13/0.1.2/sbt-ide-settings-0.1.2.pom
[error]     not found: https://repo.typesafe.com/typesafe/ivy-releases/org.jetbrains/sbt-ide-settings/0.1.2/ivys/ivy.xml
[error]     not found: https://repo.scala-sbt.org/scalasbt/ivy-snapshots/org.jetbrains/sbt-ide-settings/0.1.2/ivys/ivy.xml
[trace] Stack trace suppressed: run last *:coursierSbtClassifiersResolution for the full output.
[error] (*:coursierSbtClassifiersResolution) java.lang.Exception: Encountered 1 error(s) in dependency resolution
[error] Total time: 3 s, completed 1 juil. 2016 09:53:36
>
```

copying `~/.sbt/0.13/plugins/idea-settings.sbt` to `plugin-resolvers/idea-settings.sbt` makes it work but the whole point of global plugins is to not have them defined in every project :) (also plugins for a user's favorite IDE has no place in the project anyways)

**Comment by alexarchambault** [comment]
That must originates from the default resolvers being used in `updateSbtClassifiers`. That issue was raised some time ago on gitter IIRC. Hopefully that's an easy fix (it should amount to correctly scoping a few keys when getting the resolver list in `updateSbtClassifiers`).

**Comment by alexarchambault** [comment]
Seems fixed in `1.0.0-M14`.

**Comment by jeantil** [comment]
It seems to be working indeed. Thanks
